test/test_dataloader.py

```diff
@@ -1203,6 +1203,15 @@ def test_worker_seed(self):
             seeds.add(batch[0])
         self.assertEqual(len(seeds), num_workers)
 
+    def test_worker_seed_reproducibility(self):
+        num_workers = 6
+        batch_size = 1
+        dataset = SynchronizedSeedDataset(num_workers, batch_size, num_workers)
+        dataloader = DataLoader(dataset, batch_size=batch_size, num_workers=num_workers, generator=torch.Generator().manual_seed(42))
+        actual_seeds = set(int(batch) for batch in dataloader)
+        expected_seeds = set(6909045637428952499 + i for i in range(num_workers))
```

Comment:
don't do this. seeding is only guaranteed to be consistent on the same device & version. just do two runs and compare results

Reply:
👌